### PR TITLE
feat(cicd): test install and upgrade of charts

### PR DESCRIPTION
**Pull request by paologallinaharbur**
Linked to https://github.com/newrelic/newrelic-coreint/issues/314

Fixing as well security pipeline and bumping image

### CODE REVIEW

File: .github/workflows/push_pr.yml [hunk 1]
```diff
@@ -46,6 +47,31 @@ jobs:
               echo "No unit tests found for $chart"
             fi
           done
+      - name: Setup Minikube
+        uses: manusa/actions-setup-minikube@v2.4.3
+        if: steps.list-changed.outputs.changed == 'true'
+        with:
+          minikube version: v1.20.0
```
Comment (gsanchezgavier) [inline .github/workflows/push_pr.yml:54]: nit: i think there could be some newer versions

**Review by gsanchezgavier — APPROVED** [review]
LGTM

File: .github/workflows/push_pr.yml (hunk 1 above)
Comment (paologallinaharbur) [reply .github/workflows/push_pr.yml:54]: you are right, I don't even know why we hard code that, I'll bump it or remove it. Thanks for noticing

Comment (paologallinaharbur) [reply .github/workflows/push_pr.yml:54]: we cannot remove it since it is required, but we can bump it

**Review by kang-makes — APPROVED** [review]
LGTM

**Pull request merged**